js/bybit.js

```diff
@@ -814,6 +1040,14 @@ module.exports = class bybit extends Exchange {
                 'defaultNetworks': {
                     'USDT': 'TRC20',
                 },
+                'intervals': {
```

Comment:
why are we calling it `intervals` instead of `timeframes` ?

Reply:
We still use timeframe in ohlcv functions. The interval is only for openInterest.